R/parameters.R

```diff
@@ -159,6 +167,15 @@ read_interval_pmf <- function(path,
     query <- paste(query, "AND geo_value = ?")
     parameters <- c(parameters, list(group))
   }
+  if (parameter == "right_truncation") {
+    query <- paste(query, "AND (
```

Comment:
Oh this is a good catch. Yeah this would error if we ever ran that combination. 

We don't do that here because right-truncation is always geo-specific but let me fix and add a test.

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/71/commits/07c68566d6c6fcce76f3f1b5f460ffcad00a83ec